ytgov/sfa-client
# Enhancement: Add Education Section to Student Application Endpoint Response

Relates to https://github.com/icefoganalytics/student-financial-aid/issues/2
Fixes https://github.com/icefoganalytics/sfa-client/issues/24

# Context

As part of the "view submitted applications and their current status" project, this task covers returning the education data. 

# Data Format

```json
{
  "id": "some-application-id",
  "education": {
     "educationHistory": [{ "leftHighSchool": "2023/02", "school": 310 }]
   },
}
```

# Testing Instructions

1. Boot the `sfa-client` back-end via `API_PORT=3100 db up`
2. Boot the `student-financial-aid` back-end via
```
cd src/api
npm run start
```
3. Boot the `student-financial-aid` front-end via
```
cd src/web
npm run start
```
4. Log in to the app at http://localhost:8080/
5. Check for the education history in the data blob returned by `http://localhost:3000/api/portal/students/250/applications/4299`. It will be an empty blob.
6. Check for education history in http://localhost:3000/api/portal/students/1/applications/2198. It will have some school and date info.
7. TODO: add more examples
